devops/pypi-release-new.yml

```diff
@@ -0,0 +1,102 @@
+# Simplified PyPI release pipeline
+
+# At queue time, the user selects a Test or Production deployment. The following stages
+# then Run
+# - Predeployment validation (run a set of tests against the repository)
+# - Creates a wheel and stores in Pipeline Artifact
+# - Download wheel file from Artifact, pip install, and run tests
+# - Upload the wheel to PyPI (Test or Production as specified at queue time)
+# - Install from PyPI and run tests
+
+parameters:
+- name: releaseType
+  displayName: Release Type
+  type: string
+  default: Test
+  values:
+  - Test
+  - Production
+
+variables:
+  poolImage: "ubuntu-latest"
+  poolPythonVersion: 3.6
+  # Following is actually the name of the service connection from the Project Settings page
+  kvSubscription: "Fairness - Automation (cecafb73-04ae-4432-9f96-d96925d28058)"
+  kvVaultName: fairlearndeploy
+  ${{ if eq(parameters.releaseType, 'Test')}}:
```

Comment:
I thought the point was to get rid of pypi Test deployments?

Reply:
This new pipeline can deploy to either independently. The old pipeline required a Test-PyPI deployment before going to Prod.